src/components/atoms/SfArrow/SfArrow.html

```diff
@@ -0,0 +1,10 @@
+<div class="sf-arrow">
+  <button v-on="$listeners" class="sf-arrow__button" type="button">
```

Comment:
Why not to make button a root element?

Reply:
Yup, for me is it important. Because long button have different size on default and hover state.